src/containers/audio-rewards-page/WalletModal.tsx

```diff
@@ -186,6 +201,8 @@ const ModalContent = ({
   // TODO: user models need to have wallets
   const wallet = account.wallet as WalletAddress
 
+  const solWallet: string = (account as any).userBank?.toString()
```

Comment:
userbank is of type PublicKey which can be imported from solana web3, but for simplicity and to reduce deps because it is of common type user which is exported to mobile, will just store as string.